client/src/components/requester_order/partial-complete.html

```diff
@@ -1,16 +1,16 @@
 <div id="requester-order-complete">
-  <h1> Queue hero to the rescue! </h1>
-
-  <div>
-    <div>Meet {{ ctrl.order.queueHero }}</div>
-    <img src="" alt="Queue Hero Pic">
-    <div>
+  <div class="container">

```

Comment:
In bootstrap, you are not allowed to nest `container`. Having multiple containers will cause issues with the margins and paddings. [See Here](http://getbootstrap.com/css/#overview-container)